src/components/ProfileSettings/SettingsComponent/AccountManagementComponent/AccountManagement.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+import './AccountManagement.css';
+import { useState } from 'react';
+import InputComponent from '@/components/SharedComponents/InputComponent';
+import { useRouter } from 'next/navigation';
+import Image from 'next/image';
+
+interface SettingsAccountManagementItem {
+	//rules
+	title: 'Account' | 'Notifications';
+	href?: string; //optional parameter, question mark is optional
+	src: string;
+	src2: string;
+}
+
+const AccountManagementComponent = ({
+	title,
+	href,
+	src,
+	src2,
+}: SettingsAccountManagementItem) => {
+	//using rules and creating functions
+	const router = useRouter(); //refresh user to refresh page
+	const clickhandle = () => {
+		if (href) {
+			router.push(href);
+		}
+	};
+
+	return (
+		<section onClick={clickhandle}>
+			<div className="horizontal-line mx-auto flex">
+				<Image
```

Comment:
Can use SVG's here rather than images as Hashem mentioned. 

https://heroicons.com/
Account: Search `Account` 
Settings: Search `gear` & ones with `tooth` are nice
Arrow: Search `arrow` 

Utilize CSS / Tailwind once you use these SVG's to resize the components / elements